docs/marionette.application.md

```diff
@@ -130,11 +130,11 @@ var App = Mn.Application.extend({
     console.log('Initialize');
   },
 
-  onBeforeStart: function(options) {
-    this.model = new MyModel(options.data);
+  onBeforeStart: function(app, options) {
+    this.model = new MyModel(this.options.data);
```

Comment:
@scott-w  Please note currently on lines 58, 133 and 137 having options as the first argument is misleading since app is passed first so this is why I would prefer these lines either include both arguments or not include either since they are not being used.